src/lib/constants/toastMessage.ts

```diff
@@ -2,3 +2,9 @@ export const creaetListToastMessage = {
   uploadImageError: '이미지를 업로드 하는 중에 오류가 발생했어요. 다시 업로드해주세요.',
   createListError: '리스트 생성에 실패했어요. 다시 시도해주세요.',
 };
+
+export const editProfileToastMessage = {
+  editProfileSuccess: '프로필을 수정했습니다.🥰',
+  editProfileError: '프로필 수정에 실패했어요. 다시 시도해주세요.🥲',
+  imageSizeError: '사진이 너무 커요. 5MB 이하 사진을 넣어주세요.🥹',
+};
```

Comment:
이모티콘 넘 기엽네요 서영님😊 
그리고 토스트에러메세지 넣으면서 생각해본건데 다국어기능 지원을 위해 처음부터 아래처럼 ko와 en을 나누어 상수화해도 좋을 것 같다는 생각이 들었습니다, 그렇게되면 같은 Key를 사용할 수 있고, 언어부분(ko/en)만 객체의 키로 접근하거나, 파라미터로 보내주면 더 간결하게 다국어를 지원할 수 있을 것 같습니다.

```
export const toastMessage = {
  ko: {
    requiredLogin: '로그인이 필요해요.',
    limitFollow: `최대 ${MAX_FOLLOWING.toLocaleString('ko-KR')}명까지 팔로우할 수 있어요.`,
    editProfileSuccess: '프로필을 수정했습니다.🥰',
    editProfileError: '프로필 수정에 실패했어요. 다시 시도해주세요.🥲',
    imageSizeError: '사진이 너무 커요. 5MB 이하 사진을 넣어주세요.🥹',
  },
  // 언어변경 시, en 추가
  en: {
    editProfileSuccess: 'abcd',
    editProfileError: '...',
    imageSizeError: '...',
  },
};

```

Reply:
감사합니다 소현님😻 이모지 넣는 것이 괜찮을지 논의가 필요한데, 우선은 넣어두었어요!ㅎㅎ 

정말 좋은 생각인 것 같습니다! 우선 아래와 같이 수정해두었습니다! (gpt야 번역 고마워~)

```suggestion
const toastMessage = {
  ko: {
    uploadImageError: '이미지를 업로드에 실패했어요. 다시 업로드해주세요.🥲',
    createListError: '리스트 생성에 실패했어요. 다시 시도해주세요.🥲',
    updateProfileSuccess: '프로필을 수정했습니다.🥰',
    updateProfileError: '프로필 수정에 실패했어요. 다시 시도해주세요.🥲',
    imageSizeError: '사진이 너무 커요. 50MB 이하 사진을 넣어주세요.🥹',
  },
  en: {
    uploadImageError: 'Failed to upload the image. Please try again.🥲',
    createListError: 'Failed to create the list. Please try again.🥲',
    updateProfileSuccess: 'Profile updated successfully.🥰',
    updateProfileError: 'Failed to update the profile. Please try again.🥲',
    imageSizeError: 'The image is too large. Please insert an image smaller than 50MB.🥹',
  },
};

export default toastMessage;

```